src/web/src/components/DepartmentList.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="d-flex flex-column flex-grow-1">
+    <div
+      v-for="(major, index) in majors"
+      :index="index"
+      :key="major"
+      role="tablist"
+    >
+      <template>
+        <div class="p-2 mb-1 w-100">
+          <b-button
+            squared
+            v-b-toggle="id + 'accordion-' + index"
```

Comment:
should this be `:v-b-toggle`?

Reply:
It actually doesn't work when you add that, I'm not sure why. This was just the way that it was in the vue docs